build.yaml

```diff
@@ -427,9 +427,9 @@ common_cli:
 # flocker.node.functional is hanging, so we don't run it
 job_type:
   run_trial:
-  # http://build.clusterhq.com/builders/flocker-centos-7
+    # http://build.clusterhq.com/builders/flocker-centos-7
```

Comment:
Is there some kind of formatter / standard used here which people without vi could use.

I ask because before we had `flake8` tests on our Python we had horrible diffs when e.g. someone's editor formatted things and a reviewer had to manually discern what was a relevant change and what wasn't. Again, something to consider but not address now.